includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -230,13 +230,17 @@ function get_sortable_columns() {
 	public function prepare_items() {
 		// Handle orderby.
 		$orderby = '';
+		// phpcs:ignore WordPress.Security.NonceVerification
```

Comment:
Notice that this class was deprecated in 4.3.0, so I decided to just ignore the issues here.